fsspec/spec.py

```diff
@@ -1455,7 +1459,7 @@ def _upload_chunk(self, final=False):
         # may not yet have been initialized, may need to call _initialize_upload
 
     def _initiate_upload(self):
-        """ Create remote file/upload """
```

Comment:
Not clear why some of the docstrings got re-formatted, but others didn't. Probably some pre-commit hooks your your editor? Just looks like unrelated uncomplete changes, better to avoid.